browser/src/Editor/NeovimSurface.tsx

```diff
@@ -59,6 +60,12 @@ export class NeovimSurface extends React.PureComponent<INeovimSurfaceProps, {}>
                 <div className="stack layer">
                     <TypingPrediction typingPrediction={this.props.typingPrediction}/>
                     <Cursor typingPrediction={this.props.typingPrediction}/>
+                    <CommandLine
```

Comment:
You may not need to specify the `props` here anymore - since `CommandLine` is a connected component, it'll get these from the store via `mapStateToProps`.

Reply:
This is an issue with how I did mapstatetoprops maybe? I kept getting an error when they weren't there and I'm not 100% sure why